server/app/services/memory_store.rb

```diff
@@ -0,0 +1,55 @@
+# Simple memory store for Rack::Attack throttling
+class MemoryStore
```

Comment:
Or even easier, [ActiveSupport::Cache::MemoryStore](http://apidock.com/rails/v4.2.7/ActiveSupport/Cache/MemoryStore) since we already have active support in gems.